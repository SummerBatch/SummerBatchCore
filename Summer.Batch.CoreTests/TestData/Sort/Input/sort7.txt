02974536654215120087