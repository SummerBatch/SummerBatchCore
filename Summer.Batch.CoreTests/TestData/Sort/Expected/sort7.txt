00021215364245658797